client/src/BidList.js

```diff
@@ -1,18 +1,19 @@
 import React, { useState, useEffect } from "react";
 import { useParams, useNavigate } from "react-router-dom";
 import { get, post } from "./TenderClient";
+import "./BidList.css";
 
 const BuyerTenderList = () => {
 	const { pageNumber } = useParams();
 	const { tenderId } = useParams();
 	const currentPage = pageNumber ? parseInt(pageNumber, 10) : 1;
-	const [bid, setBid] = useState([]);
-	const [updateStatus, setUpdatedStatus] = useState({
-		id: 1,
-		status: "Active",
-	});
+	const [bids, setBids] = useState([]);
+	const [updateStatus, setUpdatedStatus] = useState("");
 	const [errorMsg, setErrorMsg] = useState(null);
+	const [statusError, setStatusError] = useState(null);
+	const [isVisible, setIsVisible] = React.useState(false);
```

Comment:
This is unnecessary, just set `updateStatus` to an empty string after the timeout